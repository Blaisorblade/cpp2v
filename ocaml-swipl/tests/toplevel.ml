open Swipl
open Embedding

let _ =
  initialise [|Sys.argv.(0); "-q"|];
  Printf.printf "Running toplevel.\n%!";
  match toplevel () with
  | true -> Printf.printf "Successfully ran the toplevel!\n%!"
  | _    -> Printf.printf "Failed to run the toplevel!\n%!"
