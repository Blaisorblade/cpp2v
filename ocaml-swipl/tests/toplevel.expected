Running toplevel.

true.


Successfully ran the toplevel!
